src/containers/orgs/components/DeleteOrgModal.js

```diff
@@ -0,0 +1,188 @@
+/*
+ * @flow
+ */
+
+import React, { Component } from 'react';
+
+import styled from 'styled-components';
+import { faTrashAlt } from '@fortawesome/pro-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Map } from 'immutable';
+import { OrganizationsApiActions } from 'lattice-sagas';
+import {
+  Button,
+  Input,
+  Modal,
+  Spinner,
+} from 'lattice-ui-kit';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { RequestStates } from 'redux-reqseq';
+import type { RequestSequence, RequestState } from 'redux-reqseq';
+
+const { DELETE_ORGANIZATION } = OrganizationsApiActions;
+
+const DeleteButton = styled(Button)`
+  align-self: center;
+  margin: 80px 0 50px 0;
+  width: 300px;
+
+  svg {
+    margin-right: 8px;
+  }
+`;
+
+const MinWidth = styled.div`
+  min-width: 500px;
+`;
+
+type Props = {
+  actions :{
+    deleteOrganization :RequestSequence;
+  };
+  org :Map;
+  requestStates :{
+    DELETE_ORGANIZATION :RequestState;
+  };
+};
+
+type State = {
+  deleteConfirmationText :string;
+  isValidConfirmation :boolean;
+  isVisible :boolean;
+}
+
+class DeleteOrgModal extends Component<Props, State> {
+
+  constructor(props :Props) {
+
+    super(props);
+    this.state = {
+      deleteConfirmationText: '',
+      isValidConfirmation: true,
+      isVisible: false,
+    };
+  }
+
+  handleOnChangeDeleteConfirmation = (event :SyntheticInputEvent<HTMLInputElement>) => {
+
+    this.setState({
+      deleteConfirmationText: event.target.value || '',
+      isValidConfirmation: true,
+    });
+  }
+
+  delete = () => {
+
+    const { actions, org } = this.props;
+    const { deleteConfirmationText } = this.state;
+
+    if (deleteConfirmationText === 'delete') {
```

Comment:
Not critical but what about typing the name of the org?

Reply:
Done.